src/cmd/linuxkit/run_vmware.go

```diff
@@ -151,15 +183,15 @@ func buildVMX(cpus int, mem int, diskPath string, prefix string) string {
 		returnString += fmt.Sprintf(vmxCdrom, cdromPath)
 	}
 
-	if diskPath != "" {
-		returnString += fmt.Sprintf(vmxDisk, diskPath)
+	vmdkPath := prefix + ".vmdk"
+	if _, err := os.Stat(vmdkPath); os.IsNotExist(err) {
+		log.Fatalf("File [%s] does not exist in current directory", vmdkPath)
 	} else {
```

Comment:
same on the `else` case here: I think we should distinguish between `err != nil && !os.IsNotExist(err)` vs. `err == nil`